tools/parabricks/all_fasta.loc.sample

```diff
@@ -0,0 +1,18 @@
+#This file lists the locations and dbkeys of all the fasta files
```

Comment:
This is not what you want I think.
You would like the user to be able to select from pre-built indices for `bwa mem`, no?
So the .loc.sample file to include here is either `bwa_mem_index.loc.sample` or `bwa_mem2_index.loc.sample` (don't know which one fq2bam is supposed to be compatible with.
Also, this file needs to go into a tool-data subfolder.